pkg/ls/fileinfo_tamago.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
```

Comment:
I need to learn a bit about tamago, but I think build tags are missing
```
//go:build tamago
```

Reply:
no, since the file ends in _tamago, that is an implicit build tag, as with files that end in _linux, etc.
GOOS=tamago etc. etc.

Reply:
yeah, upstreaming tamago will solve the build issue, but adding build tag will probably fix build failure a bit faster :)

Reply:
fixed